draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -644,6 +645,7 @@ GET http://www.example.org/pub/WWW/TheProject.html HTTP/1.1
 <sourcecode type="http-message">
 CONNECT www.example.com:80 HTTP/1.1
 Host: www.example.com
+
 </sourcecode>
```

Comment:
The full request message, of course, includes the blank line to end the headers section.
Whether it is worth trying to be pedantically correct about this across all the `<sourcecode>` elements is not entirely clear, though, given that the presentation form of it is unlikely to show the blank line in a meaninful way.